Stripe.podspec

```diff
@@ -6,11 +6,11 @@ Pod::Spec.new do |s|
   s.homepage                       = 'https://stripe.com/docs/mobile/ios'
   s.authors                        = { 'Jack Flintermann' => 'jack@stripe.com', 'Stripe' => 'support+github@stripe.com' }
   s.source                         = { :git => 'https://github.com/stripe/stripe-ios.git', :tag => "v#{s.version}" }
-  s.frameworks                     = 'Foundation', 'Security'
+  s.frameworks                     = 'Foundation', 'Security', 'WebKit'
```

Comment:
Probably. It is unclear to me why PassKit and AddressBook are listed as weak frameworks instead of required. Any thoughts on that?